forge/routes/api/admin.js

```diff
@@ -449,18 +449,26 @@ module.exports = async function (app) {
         if (recipientRoles && !recipientRoles.every(value => Object.values(Roles).includes(value))) {
             return reply.code(400).send({ code: 'bad_request', error: 'Invalid Role provided.' })
         }
+        let teamTypes
+        if (filter?.teamTypes && filter.teamTypes.length > 0) {
+            teamTypes = filter.teamTypes.map(app.db.models.TeamType.decodeHashid).flat()
+        }
+        let billing
+        if (filter?.billing && filter.billing.length > 0) {
+            billing = filter.billing
+        }
         if (mock) {
             // If mock is sent, return an indication of how many users would receive this notification
             // without actually sending them.
-            const count = await app.db.models.User.byTeamRole(recipientRoles, { summary: true, count: true })
+            const count = await app.db.models.User.byTeamRole(recipientRoles, { teamTypes, billing, summary: true, count: true })
             reply.send({
                 mock: true,
                 recipientCount: count
             })
             return
         }
 
-        const recipients = await app.db.models.User.byTeamRole(recipientRoles, { summary: true })
+        const recipients = await app.db.models.User.byTeamRole(recipientRoles, { teamTypes, summary: true })
```

Comment:
```suggestion
        const recipients = await app.db.models.User.byTeamRole(recipientRoles, { teamTypes, billing, summary: true })
```

Reply:
I assume billing was meant to be passed along to the subsequent call

Reply:
Hmm, yes. Let me revisit as my testing locally definitely covered this...